src/util/command_line_handling.cpp

```diff
@@ -1011,7 +1011,7 @@ namespace hpx { namespace util
 #endif
 
         // handle setting related to schedulers
-        queuing_ = detail::handle_queuing(cfgmap, vm, "local-priority-fifo");
+        queuing_ = detail::handle_queuing(cfgmap, vm, "local-workstealing-fifo");
```

Comment:
Should we already change the default? I suppose the answer depends on the performance results... And should we add the new scheduler to the suspension tests?

Reply:
This is done on this branch only to run all tests using the new scheduler. Before merging I was planning to revert this change.

Reply:
Leaving this unresolved just so we don't forget to change this back.